lib/configure.js

```diff
@@ -280,6 +280,43 @@ function configure (gyp, argv, callback) {
       }
     }
 
+    // For z/OS we need to set up the path to zoslib include directory,
+    // which contains headers included in v8config.h.
+    var zoslibIncDir
+    if (process.platform === 'os390') {
+      logprefix = "find zoslib's zos-base.h:"
+      let msg
```

Comment:
Should `let` be used here? It seems like the convention for this file is entirely `var` and `const`.

Reply:
using var will cause linter to complain as msg was defined in line 277